docs/installation/installing.rst

```diff
@@ -1,4 +1,86 @@
 .. _install_overview:
 
 Installing KSQL
-===============
\ No newline at end of file
+===============
+
+KSQL is a component of |cp| and the KSQL binaries are located at `https://www.confluent.io/download/ <https://www.confluent.io/download/>`_
+as a part of the |cp| bundle.
+
+KSQL must have access to a running Kafka cluster, which can be on prem, |ccloud|, etc.
```

Comment:
nit: 'on prem' seems like internal jargon. maybe "which can be in your data center, in a public cloud, in |ccloud|, etc"